tests/smoke.runner.js

```diff
@@ -347,6 +347,45 @@ const jasmineSpecFiltering = async () => {
     assert.strictEqual(skippedSpecs, 2)
 }
 
+/**
+ * Mocha with spec grouping feature enabled
+ */
+const mochaSpecGrouping = async () => {
+    const { skippedSpecs } = await launch(
+        path.resolve(__dirname, 'helpers', 'config.js'),
+        {
+            specs: [
+                [
+                    path.resolve(__dirname, 'mocha', 'test-empty.js'),
+                    path.resolve(__dirname, 'mocha', 'test-skipped.js'),
+                    path.resolve(__dirname, 'mocha', 'test-skipped-grep.js')
+                ]
+            ]
+        })
+    // Specs will be treated as a group, so no specs will be skipped
+    assert.strictEqual(skippedSpecs, 0)
+}
+
+/**
+ * Jasmine with spec grouping feature enabled
+ */
+const jasmineSpecGrouping = async () => {
+    const { skippedSpecs } = await launch(
+        path.resolve(__dirname, 'helpers', 'config.js'),
+        {
+            specs: [
+                [
+                    path.resolve(__dirname, 'jasmine', 'test.js'),
+                    path.resolve(__dirname, 'jasmine', 'test-skipped.js'),
+                    path.resolve(__dirname, 'jasmine', 'test-skipped-grep.js')
+                ]
+            ],
+            framework: 'jasmine'
+        })
+    // Specs will be treated as a group, so no specs will be skipped
+    assert.strictEqual(skippedSpecs, 0)
+}
```

Comment:
This feature is independent from which framework is being run, so we can remove one smoke test
```suggestion
```

Reply:
Yes agreed